apps/video/js/view.js

```diff
@@ -504,6 +504,22 @@
     }
   }
 
+  function seekEnded() {
+    updateSlider();
+    if (pendingSeekPosition) {
```

Comment:
pendingSeekPosition is undeclared here.  For stupid historical reasons, the video app has two completely separate video player implementaions.  The main entry point is index.html and that uses the video player in video.js.  But for the view activity, the entry point is view.html, which loads view.js but does not load video.js.  So you've got to declare this variable.

There should have been a lint warning about the undeclared variable.  If you've run 'npm install' and have the linter code installed, then you should not have been able to commit this code, because it would have failed the linter unless you either declared the variable or added it to the /* global */ comment at the top...